### ISSUE TITLE
Weird string " Channel reop " in geoip whois line on unrealircd

### ISSUE THREAD

**Issue by d3xt3r01** [weechat/weechat#1783]
## Bug summary
Doing a whois on somebody shows a " Channel reop " string on the geoip line

## Steps to reproduce

1.  Connect to unrealircd 6.0.3
2. whois somebody

## Current behavior

The line with the geoip data has some " Channel reop " text prepended

## Expected behavior

Just the information after the weird prepended text.

```
19:24:18                    --> d3x | :irc.d3x 311 mynick gst TEST Clk-83FCA2D4 * :...
19:24:18                    --> d3x | :irc.d3x 379 mynick gst :is using modes +ipwxzT
19:24:18                    --> d3x | :irc.d3x 378 mynick gst :is connecting from *@... ...
19:24:18                    --> d3x | :irc.d3x 319 mynick gst :#romania
19:24:18                    --> d3x | :irc.d3x 312 mynick gst irc.d3x :d3x Server
19:24:18                    --> d3x | :irc.d3x.ro 671 mynick gst :is using a Secure Connection [TLSv1.2-ECDHE-RSA-AES256-GCM-SHA384]
19:24:18                    --> d3x | :irc.d3x.ro 320 mynick gst :is using an IP with a reputation score of 211
19:24:18                    --> d3x | :irc.d3x.ro 344 mynick  gst ES :is connecting from Spain
19:24:18                    --> d3x | :irc.d3x.ro 317 mynick gst 16124 1652914189 :seconds idle, signon time
19:24:18                    --> d3x | :irc.d3x.ro 318 mynick gst :End of /WHOIS list. 
```
```
19:24:18                    d3x  -- | [gst ] (TEST@Clk-83FCA2D4): ...
19:24:18                    d3x  -- | [gst ] is using modes +ipwxzT
19:24:18                    d3x  -- | [gst ] is connecting from *@... ...
19:24:18                    d3x  -- | [gst ] #romania
19:24:18                    d3x  -- | [gst ] irc.d3x (d3x Server)
19:24:18                    d3x  -- | [gst ] is using a Secure Connection [TLSv1.2-ECDHE-RSA-AES256-GCM-SHA384]
19:24:18                    d3x  -- | [gst ] is using an IP with a reputation score of 211
19:24:18                    d3x  -- | Channel reop gst : ES is connecting from Spain
19:24:18                    d3x  -- | [gst ] idle: 04 hours 28 minutes 44 seconds, signon at: Thu, 19 May 2022 01:49:49
19:24:18                    d3x  -- | [gst ] End of /WHOIS list.
```

---

- WeeChat version:  3.4.1
- OS, distribution and version:  gentoo
- Terminal: xterm / xfce4-terminal
- Terminal multiplexer (screen/tmux/…/none):  tmux

**Comment by flashcode** [comment]
Hi,

This issue is supposed to be fixed in WeeChat 3.5 via #1736, could you please try and confirm?
If confirmed, this issue can be closed.

Thanks.

**Comment by d3xt3r01** [comment]
Yup. Can confirm it's fixed.
